lona/server.py

```diff
@@ -102,8 +102,9 @@ def __init__(self, app, project_root, settings_paths=None,
 
         # setup server state
         server_logger.debug('setup server state')
-
-        self._state = ServerState(initial_data={})
+        self._state = ServerState(
+            initial_data=self.settings.INITIAL_STATE,
```

Comment:
We cant just pass the original settings variable in here because this way you could change ``server.state`` using ``server.settings.INITIAL_STATE`` and bypass all locks. Please use deepcopy.

Reply:
Fixed.
And also followed your refactoring from ``ServerState`` to ``State``.